test/nft-collection-list/nft-collection-list.test.ts

```diff
@@ -45,4 +50,40 @@ describe('NFTCollectionList', function () {
     expect(mainnetJson.networkId).toEqual(0)
     expect(testnetJson.networkId).toEqual(1)
   })
+
+  it('validate nft collection types', () => {
+    return Promise.all([
+      validateNftType(mainnetNFTCollectionList, mainnetURL),
+      validateNftType(testnetNFTCollectionList, testnetURL)
+    ])
+  })
+
+  it('validate nft collection metadata', () => {
+    return Promise.all([
+      validateNftMetadata(mainnetNFTCollectionList, mainnetURL),
+      validateNftMetadata(testnetNFTCollectionList, testnetURL)
+    ])
+  })
+
+  async function validateNftType(nftList: NFTCollectionList, url: string) {
```

Comment:
Maybe also update `validateNftType` and `validateNftMetadata` to be `validateNftCollectionType` and `validateNftCollectionMetadata`